packages/components/src/tooltip/test/index.js

```diff
@@ -97,37 +97,31 @@ describe( 'Tooltip', () => {
 				</Tooltip>
 			);
 
-			const button = wrapper.find( 'button' );
-
-			let event;
-
-			event = { type: 'mousedown' };
-			button.simulate( event.type, event );
+			const button = screen.getByRole( 'button' );
+			fireEvent.mouseDown( button );
 			expect( originalOnMouseDown ).toHaveBeenCalledWith(
 				expect.objectContaining( {
-					type: event.type,
+					type: 'mousedown',
 				} )
 			);
 
-			event = { type: 'focus', currentTarget: {} };
-			button.simulate( event.type, event );
-
-			const popover = wrapper.find( 'Popover' );
+			button.focus();
+			const popover =
+				container.getElementsByClassName( 'components-popover' );
 			expect( popover ).toHaveLength( 0 );
 
-			event = new window.MouseEvent( 'mouseup' );
-			await act( async () => document.dispatchEvent( event ) );
+			fireEvent.mouseUp( button );
 			expect( originalOnMouseUp ).toHaveBeenCalledWith(
 				expect.objectContaining( {
-					type: event.type,
+					type: 'mouseup',
 				} )
 			);
 		} );
 
 		it( 'should show popover on delayed mouseenter', () => {
```

Comment:
We should try to simplify this test by:

- using `@testing-library/user-event` and its `hover` utility function
- using testing library's `findBy` or `waitFor` utilities to select an element in the page asynchronously

Reply:
Fixed by 10274ad

The original implementation was to wait for the time indicated by the constant (`TOOLTIP_DELAY`) and then check for the popover to appear.
I simply wait for the popover text to appear by the `findBy***` method.
Should I strictly wait the time specified in `TOOLTIP_DELAY`?

Reply:
This test removed by c8b1c0a.
See: https://github.com/WordPress/gutenberg/pull/43061#discussion_r949393174